operators/view_history.py

```diff
@@ -39,6 +39,22 @@ def execute(self, context):
         for prop in selection.__annotations__.keys():
             if hasattr(context.scene.dream_textures_prompt, prop):
                 setattr(context.scene.dream_textures_prompt, prop, getattr(selection, prop))
+            # when the seed of the promt is found in the available image datablocks, use that one in the open image editor
+            # note: when there is more than one image with the seed in it's name, do nothing. Same when no image with that seed is available.
+            if prop == 'seed':
+                seed_string = str(getattr(selection, prop))
+                image_names_no_suffix = [i.name.split('.')[0] for i in bpy.data.images]
+                image_names_no_suffix_seed_count = image_names_no_suffix.count(seed_string)
+                if image_names_no_suffix_seed_count == 1:
+                    print("Image for seed " + seed_string + " is unique. Selecting image datablock.")
+                    for area in context.screen.areas:
+                        if area.type != 'IMAGE_EDITOR':
+                            continue
+                        area.spaces.active.image = bpy.data.images[seed_string]
+                elif image_names_no_suffix_seed_count > 1:
+                    self.report({'INFO'}, "More than one image found for seed " + seed_string + ". Not changing image editor contents.")
+                elif image_names_no_suffix_seed_count == 0:
+                    self.report({'INFO'}, "No image found for seed " + seed_string + ". Not changing image editor contents.")
```

Comment:
I think I would rather show no message if an existing image isn't found, because users may think this is an error. I think it may be common to try reloading a prompt from a file that isn't open, so it may error more often than it doesn't.
```suggestion
```